src/editor/components/modals/PaymentModal/PaymentModal.component.jsx

```diff
@@ -23,7 +23,7 @@ const PaymentModal = ({ isOpen, onClose }) => {
   const { currentUser } = useAuthContext();
   const [isLoading, setIsLoading] = useState(false);
 
-  if (currentUser?.isPro) {
+  if (location.pathname.includes('success')) {
```

Comment:
You want to check for success in `/#/modal/payment/success` url right?
That won't work. All the part after the # is not part of location.pathname but part of location.hash
You want location.href or location.hash here.